azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -122,6 +122,9 @@ public void uploadExecutableNode(ExecutableNode node, Props inputParams)
   @Override
   public void updateExecutableNode(ExecutableNode node)
       throws ExecutorManagerException {
+    if (!nodes.containsKey(node.getId())) {
+      uploadExecutableNode(node, new Props());
+    }
```

Comment:
It seems that this error happened consistently. It would be good to find out if this is an indication of a real issue.